source/content/guides/drupal/drupal-hosted-createempty-md/01-introduction.md

```diff
@@ -1,32 +1,32 @@
 ---
```

Comment:
@stovak , still contains multiple references to D9 and D8.  See preview link: @stovak , still contains multiple references to D9.  See preview link: